hooks/tk-multi-publish2/basic/publish_image.py

```diff
@@ -168,45 +161,52 @@ def validate(self, settings, item):
         # get the normalized path
         path = sgtk.util.ShotgunPath.normalize(path)
 
-        # ensure the publish template is defined and valid and that we also have
-        publish_template = publisher.get_template_by_name(template_name)
-        if not publish_template:
-            self.logger.error(
-                "The valid publish template could not be determined for the "
-                "export image item."
+        if template_name:
+            # ensure the publish template is defined and valid and that we also have
+            publish_template = publisher.get_template_by_name(template_name)
+            if not publish_template:
+                self.logger.error(
+                    "The valid publish template could not be determined for the "
+                    "export image item."
+                )
+                return False
+
+            item.local_properties.publish_template = publish_template
+
+            # get the configured work file template
+            work_template = item.parent.properties.get("work_template")
```

Comment:
hey @julien-lang , here I'll add an extra test to make sure the work template exist as well
you could have a weird behavior where someone added a publish template in the settings but no work template (or a bad one) in the parent item
(basically, moving here the test that was done before in the accept() method)

Reply:
Done! Thanks!